test/helpers/UniStaker.handler.sol

```diff
@@ -157,7 +157,8 @@ contract UniStakerHandler is CommonBase, StdCheats, StdUtils {
   function claimReward(uint256 _actorSeed) public countCall("claimReward") doCheckpoints {
     _useActor(_beneficiaries, _actorSeed);
     vm.startPrank(_currentActor);
-    uint256 rewardsClaimed = uniStaker.unclaimedRewardCheckpoint(_currentActor);
+    uint256 rewardsClaimed =
+      uniStaker.scaledUnclaimedRewardCheckpoint(_currentActor) / uniStaker.SCALE_FACTOR();
     uniStaker.claimReward();
     vm.stopPrank();
```

Comment:
(arbitrary thread)

I looked at one of the failing tests and it seems like `testFuzz_CalculatesCorrectEarningsForFourUsersWhenTwoStakeMorePartiallyThroughTheDurationAndOneBeneficiary` is failing because the `DelegateSurrogate` is `_depositor1`. It may be similar for the other cases

Reply:
Unclear what changed with Foundry that these failures started. Issue submitted here: https://github.com/foundry-rs/foundry/issues/7462

Workaround implemented and merge here: https://github.com/uniswapfoundation/UniStaker/commit/157345110ec3d22e32687d0f6812f9f33cd9f4e7